asteroid/masknn/attention.py

```diff
@@ -64,8 +58,7 @@ def forward(self, x):
         # mha is seq_len, batch, channels
         # self-attention is applied
         out = self.mha(tomha, tomha, tomha)[0]
-        out = self.pos_wise_ff(out.permute(1, 0, 2)).transpose(1, -1)
-        x = self.dropout(out) + x
+        x = self.dropout(out.permute(1, 2, 0)) + x
```

Comment:
I'm curious whether the dropouts at line 61, 65, 66 shared the same self.dropout would cause any problem?

Reply:
It's a wrapper above a function and has no state, so it doesn't case any problem, I asked myself the same question in the past :wink: 
```python
    def forward(self, input: Tensor) -> Tensor:
        return F.dropout(input, self.p, self.training, self.inplace)
```